mobility_games/launch/MusicalCane.launch

```diff
@@ -0,0 +1,12 @@
+<launch>
```

Comment:
Can you add the following to the top of the file?  It makes the launch file have color coding in my text editor
```
<?xml version="1.0"?>
```